src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -13,104 +13,103 @@
  */
 public class CommandResult {
 
-    private final String feedbackToUser;
+    /** Feedback to be shown to the user through CLI. */
+    private final String cliFeedbackToUser;
 
     /** Help information should be shown to the user. */
-    private final boolean showHelp;
+    private final boolean shouldShowHelp;
 
     /** The application should exit. */
-    private final boolean exit;
+    private final boolean shouldExit;
 
     /** This command interacts directly with the GUI */
     private final boolean hasGuiInteraction;
 
-    /** Doctor selected by user through command */
-    private final Optional<Doctor> selectedDoctor;
```

Comment:
Is there any reason for making this change?

Reply:
Yeap! Realised that it is not the intended use of Optionals to be parameter types. Rather, they should be used as return types to indicate potential absense of a value.